server/sonar-server/src/main/java/org/sonar/server/plugins/StaticResourcesServlet.java

```diff
@@ -22,8 +22,10 @@
 import com.google.common.annotations.VisibleForTesting;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
+import org.sonar.api.Plugin;
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Remove this unused import 'org.sonar.api.Plugin'.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)